wrappers/nodejs/README.md

```diff
@@ -24,24 +33,41 @@ Note#1: The npm module `windows-build-tools` also works for `npm install` Node.j
 
 Note#2: for Node.js GLFW module, add `msbuild` to system PATH, e.g "`C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin`"
 
-#### Setup Ubuntu Linux 16.04 Build Environment
-
-TODO:
+Note#3: When running `Node.js` 6.x, you might need to [upgrade npm-bundled `node-gyp`](https://github.com/nodejs/node-gyp/wiki/Updating-npm%27s-bundled-node-gyp) to support Visual Studio 2017 (if you're using it)
 
-## 1.2. Build Native C++ librealsense2 ##
+## 1.2. Build Native C++ `librealsense` ##
 
 Please refer to [Linux installation doc](../../doc/installation.md) or [Windows installation doc](../../doc/installation_windows.md) to build native C++ librealsense2.
 
 ## 1.3. Build Node.js Module/Addon ##
 
+There are two options to do it: "manually bulid" or "build with CMake".
+The former one is for `Node.js` language binding developers who frequently change module/addon source code; the latter one is for all-in-one build scenarios.
+
+### Manually Build
+
+After C++ `librealsense` library is Built, run the following commands:
+
 ```
 cd wrappers/nodejs
 npm install
 ```
 
+### Build with CMake
+
+Before building C++ `librealsense` library, enable the following option when calling `cmake`.
+```
+cmake -DBUILD_NODEJS_BINDINGS=1 <other options...>
+make  # Will build both C++ library & Node.js binding.
+```
+
+
+Note: when doing "Build with CMake" on Windows, `node-gyp` module of 'npm install' command requires both `.NET Framework 4.7 development tools` & `.NET Framework 4-4.6 development tools` to be installed along with `Visual Studio 2017` (if you're using it).
```

Comment:
As I tested, .NET `4.7` does not work, my env works well with .NET `4.6.2`

Reply:
I've tested several times, `4-4.6` is for `<path-4.6.1>/Libs/um/x64` and `4.7` is for `<path-4.7>/libs/um/x64`. The first wrong impression is because of wrong UI operation like unchecking all `.NET desktop development` components and checking it back again.

Reply:
So after some discussion, the decision is to mention all the three components and let users have a try, and also mention the `GYP_MSVS_VERSION` env var which can force `gyp` to work on 2015 mode (actually 2017)

Reply:
agree, please make the change